### PR TITLE
Fixes #15292: Move /opt/rudder/etc/ssl/nodeslist.cert to /var/rudder/lib/ssl

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15292

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: relay/sources/rudder-server-relay-postinst [hunk 1]
```diff
@@ -72,8 +72,9 @@ if [ ! -f /opt/rudder/etc/ssl/ca.cert ]; then
   cp /opt/rudder/etc/ssl/rudder.crt /opt/rudder/etc/ssl/ca.cert
 fi
 
-if [ ! -f /var/rudder/ssl/nodescerts.pem ]; then
-  touch /var/rudder/ssl/nodescerts.pem
+if [ ! -f /var/rudder/lib/ssl/nodescerts.pem ]; then
+  mkdir -p /var/rudder/lib/ssl
```
Comment (amousset) [inline relay/sources/rudder-server-relay-postinst]: shouldn't this be done in the install target of the makefile?

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**